CHANGES.txt

```diff
@@ -5,6 +5,11 @@ CHANGES for Crate ReadTheDocs Theme
 Unreleased
 ==========
 
+2015/06/18 0.4.0
```

Comment:
creating a new version is always done with a dedicated commit on master, see: https://github.com/crate/crate-docs-theme/blob/master/DEVELOP.rst#distributing